index.js

```diff
@@ -18,6 +18,6 @@ module.exports = fp((fastify, opts, next) => {
 
   next()
 }, {
-  fastify: '1.x',
+  fastify: '>=1.x',
```

Comment:
I think I'd feel better about not making this quite so open-ended. What are your thoughts?